data-pipeline/src/trace_exporter.rs

```diff
@@ -256,12 +258,11 @@ mod tests {
             exporter.endpoint.url.to_string(),
             "http://192.168.1.1:8127/v0.7/traces"
         );
-        assert_eq!(builder.host.unwrap(), "192.168.1.1");
-        assert_eq!(builder.port.unwrap(), 8127);
-        assert_eq!(builder.tracer_version.unwrap(), "v0.1");
-        assert_eq!(builder.language.unwrap(), "nodejs");
-        assert_eq!(builder.language_version.unwrap(), "1.0");
-        assert_eq!(builder.interpreter.unwrap(), "v8");
+
+        assert_eq!(exporter.tags.tracer_version, "v0.1");
```

Comment:
I switched to testing the exporter tags because this test is calling `build()` which sets the exporter tags and we should be testing outputs in unit tests, unless we have a special reason not to.